<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="general_tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLineEdit" name="plant_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Plant:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="plant_number_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Alias:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="plant_note_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Note:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="owner_comboBox">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>140</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Owner:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>PlusCode:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Desc:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="plant_description_input">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="custom_tab">
    <attribute name="title">
     <string>Custom Fields</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>361</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>361</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>361</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>361</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
